### ISSUE TITLE
simVQ: rotation trick leads big commit loss

### ISSUE THREAD

**Issue by ChuxiJ** [lucidrains/vector-quantize-pytorch#177]
add rotation trick:
![image](https://github.com/user-attachments/assets/8c15642c-58b3-4638-a9b8-623836c5ea6e)

set rotation trick false
![image](https://github.com/user-attachments/assets/5b7124dd-761a-430c-87c6-cfe00edc41ba)

code:
```python
        self.quantizer = SimVQ(
            codebook_size=2**quantizer_codebook_size_exp,
            dim=in_channels,
            rotation_trick=True,
            # rotation_trick=False,
            codebook_transform=nn.Sequential(
                nn.Linear(in_channels, 2*in_channels),
                nn.ReLU(),
                nn.Linear(2*in_channels, in_channels)
            ),
            channel_first=True,
            commitment_weight=commitment_weight, # commitment_weight=1000.0
        )
```

**Comment by KdaiP** [comment]
Have the same problem, training on TTS data.

**Comment by lucidrains** [comment]
@ChuxiJ ah, perhaps i was too hasty removing the other half of the commit loss

do you want to try 1.20.11 and let me know what you see?

**Comment by lucidrains** [comment]
i think this is resolved, please reopen if not so

**Comment by zaptrem** [comment]
This appears to still be happening: 

![Image](https://github.com/user-attachments/assets/c0e56eb7-7707-4bd2-adb0-cb5e1d5d8a6b)
